.github/workflows/ci-quarkus-build-publish-image.yml

```diff
@@ -0,0 +1,143 @@
+name: Build/publish Docker image
+
+on:
+  workflow_call:
+    inputs:
+      image-name:
+        description: Name of Docker image
+        required: false
+        type: string
+      image-pack:
+        description: Docker image pack of builder-jammy-tiny, builder-jammy-base or builder-jammy-full
+        default: builder-jammy-tiny
+        required: false
+        type: string
+      java-version:
+        description: Main version of java
+        default: '21'
+        required: false
+        type: string
+      container-scan-offline-mode:
```

Comment:
denne er ikkje i bruk lengre, trur eg. var for azure container scan, no brukar me trivy sin direkte.